Changes the transfer history model to be inline with upstream table changes

fredkingham opened the pull request:
Upstream have added in fields and removed fields. This changes the
transfer history loader to save those fields

Commit:
Changes the transfer history model to be inline with upstream changes

Upstream have added in fields and removed fields. This changes the
transfer history loader to save those fields

Commit:
Merge branch 'prepayment' into transfer-history-2

davidmiller commented on plugins/admissions/models.py:
```diff
@@ -270,8 +273,11 @@ class Meta:
         'TRANS_HIST_TRANSFER_REASON'            : 'transfer_reason',
-        'CREATED_DATE'                          : 'created_datetime',
         'UPDATED_DATE'                          : 'updated_datetime',
+        'TRANS_UPDATED'                         : 'trans_updated_datetime',
```
These aren't the fieldnames it'll set them as properties and never save them - imagine we didn't test on test and check data ?

Commit:
Fixes transfer history so we save the correct fields to the models.

There was a bug in a previous commit that had the wrong field names in
the TransferHistory.UPSTREAM_FIELDS_TO_MODEL_FIELDS mapping.
